code/hud/hudets.cpp

```diff
@@ -289,7 +289,7 @@ void ai_manage_ets(object* obj)
 	}
 
 	// emergency check for ships with shields
-	if (!(obj->flags[Object::Object_Flags::No_shields])) {
+	if (!(obj->flags[Object::Object_Flags::No_shields]) && !(The_mission.ai_profile->flags[AI::Profile_Flags::Disable_ai_transferring_energy])) {
 		float shield_left_percent = get_shield_pct(obj);
 		if ( shield_left_percent < SHIELDS_EMERG_LEVEL_PERCENT ) {
```

Comment:
If you just want to control transferring energy and nothing else, then the flag should be more specifically applied to the section in lines 294-299.  The "return to normal values" section in lines 303-308 looks like it is there as a counterpart to lines 283-289.

Reply:
Ah good point, just updated, thanks!